src/custom-matchers/toBeEmpty.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+function isEmpty(instance: ReactTestInstance) {
+  if (instance.props.children instanceof Array) {
```

Comment:
We should use React.Children for iteration over children

Reply:
Are you saying we should check to see if it is instanceof React.Children? 
I don't understand